lsp-mode.el

```diff
@@ -256,16 +269,17 @@
 ;;; recursive call to init-lsp-conn-inner, although it's a bit challenging due
 ;;; to the callback nature.
 
-(defun init-lsp-conn-inner (ws conn)
+(defun lsp-init-conn-inner (ws conn)
   (send-lsp-msg (lsp-init (lsp-init-params nil nil ws nil))
```

Comment:
Yep, those are fixed now, sorry: `highlight-symbol-query-replace` had failed to replace all occurrences. I've pushed fresh changes.